openmc/data/endf.c

```diff
@@ -0,0 +1,39 @@
+#include <stdlib.h>
+
+double cfloat_endf(const char* buffer, int n)
+{
+  char arr[12]; // 11 characters plus a null terminator
+  int j = 0; // current position in arr
+  int found_significand = 0;
+  int found_exponent = 0;
+  for (int i = 0; i < n; ++i) {
+    // Skip whitespace characters
```

Comment:
Tiny nit: this comment might be better moved one line down, ie. immediately connected to the line: `if (c == ' ') continue;`